### ISSUE TITLE
`Layout/EmptyLineAfterMagicComment` should take into account license headers

### ISSUE THREAD

**Issue by postmodern** [rubocop/rubocop#11238]
While I was adding `# frozen_string_literal: true` comments to all of the files in an LGPL-3 project, `rubocop` incorrectly recommended that an extra space be added after the magic comment. However, every file in the project has the standard LGPL-3 license header comment at the top of the file already. I propose that it should be acceptable to follow a magic header comment with the license header:

## Example

```ruby
# frozen_string_literal: true
#
# <project> - <summary>
#
# Copyright (c) <year> <author> (<email>)
#
# <project> is free software: you can redistribute it and/or modify
# it under the terms of the GNU Lesser General Public License as published
# by the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
#
# <project> is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU Lesser General Public License for more details.
#
# You should have received a copy of the GNU Lesser General Public License
# along with <project>.  If not, see <https://www.gnu.org/licenses/>.
#

<ruby code follows>
```

## Expected behavior

No rubocop warnings.

## Actual behavior

```
C: [Correctable] Layout/EmptyLineAfterMagicComment: Add an empty line after magic comments.
```

## RuboCop version

`1.39.0 (using Parser 3.1.3.0, rubocop-ast 1.24.0, running on ruby 3.1.3) [x86_64-linux]`

**Comment by koic** [comment]
This seems to be a preference, so it can be dealt with by specifying excluded files to `Exclude` or `Enabled: false` in .rubocop.yml for `Layout/EmptyLineAfterMagicComment` cop. Thank you.
https://docs.rubocop.org/rubocop/1.39/configuration.html
